### ISSUE TITLE
[FEATURE] Send location to vehicle

### ISSUE THREAD

**Issue by jd1** [skodaconnect/homeassistant-myskoda#684]
**Is your feature request related to a problem? Please describe.**
I would like to send to location of calendar entries to my car.

**Describe the solution you'd like**
The skodaconnect component should offer a new service to send a location to the car.

**Describe alternatives you've considered**
I can use the app or web, but it don't like the manual process.

**Additional context**
There is an API which is used by [skoda-connect.com](https://www.skoda-connect.com):
POST https://api.skoda-connect.com/vehicles/{vehicle}/trips/steps
Body: `[{"name":"Tiergarten","location":{"latitude":52.514534,"longitude":13.3501015}}]`

**Comment by WebSpider** [comment]
@jd1 Have you moved to the MySkoda connectivity, and if so, can you achieve this using the MySkoda app?
If so, please document how you do this in the MySkoda app, so we can see if we can make HA mimic this.

**Comment by jd1** [comment]
Yes, I already switched to MySkoda. In the app, you can select maps and send a location to the vehicle.

**Comment by WebSpider** [comment]
Would you be so kind to make a few screenshots?

**Comment by jd1** [comment]
Sure, here are some screenshots. At first you select the destination and calculate the route.
![Image](https://github.com/user-attachments/assets/3a04051f-0a17-424c-bfd6-d2636da476eb)
Then you can send it to your vehicle.
![Image](https://github.com/user-attachments/assets/545b4f80-2717-45b3-baa4-afa7bd6ccceb)

**Comment by WebSpider** [comment]
I see this has been added in my v8.1.0 app as well. It appears we have the URL's in skodaconnect/myskoda#393 to start supporting this 🥳

**Comment by WebSpider** [comment]
We need the format of the POST request the MySkoda app makes. If anyone is able to supply this, we can proceed on this.

**Comment by WebSpider** [comment]
I'm looking into this now.

Would it work for you if you would use an `action` in HomeAssistant to send:

- Name of the location
- GPS coordinates of the location

I haven't found out how to influence the route or anything.

**Comment by jd1** [comment]
Sending coordinates would be fine for me.
